applications/FluidDynamicsApplication/python_scripts/fluid_solver.py

```diff
@@ -259,3 +273,162 @@ def _SetNodalProperties(self):
         # Transfer the obtained properties to the nodes
         KratosMultiphysics.VariableUtils().SetVariable(KratosMultiphysics.DENSITY, rho, self.main_model_part.Nodes)
         KratosMultiphysics.VariableUtils().SetVariable(KratosMultiphysics.VISCOSITY, kin_viscosity, self.main_model_part.Nodes)
+
+    # TODO: I THINK THIS SHOULD BE MOVED TO THE BASE PYTHON SOLVER
+    def is_restarted(self):
+        # this function avoids the long call to ProcessInfo and is also safer
+        # in case the detection of a restart is changed later
+        return self.main_model_part.ProcessInfo[KratosMultiphysics.IS_RESTARTED]
+
+    def GetEstimateDtUtility(self):
+        if not hasattr(self, '_estimate_dt_utility'):
+            self._estimate_dt_utility = self._CreateEstimateDtUtility()
+        return self._estimate_dt_utility
+
+    def GetScheme(self):
```

Comment:
```suggestion
    def _GetScheme(self):
```

those are protected functions that should not be used from outside

Reply:
OK. I did it in that way in case we require them from outside the solver in the future (honestly no use case come to my mind so I'll do it as you suggest).

Reply:
Done.